flink-table/flink-table-api-java/src/main/java/org/apache/flink/table/api/internal/BaseExpressions.java

```diff
@@ -1468,6 +1469,16 @@ public OutType mapEntries() {
         return toApiSpecificExpression(unresolvedCall(MAP_ENTRIES, toExpr()));
     }
 
+    /**
+     * Returns a map created by merging two maps, 'map1' and 'map2'. These two maps should have same
+     * data structure. If there are overlapping keys, the value from 'map2' will overwrite the value
```

Comment:
it's not clear what is same datastructure in SQL context...
i guess same type (or not same, however common)

Reply:
ok